controllers/controller.js

```diff
@@ -14,4 +14,24 @@ exports.getReviews = (req,res) => {
     .then((reviews) => {
         res.status(200).send({reviews})
     })
+}
+
+exports.getReviewsById = (req,res,next) => {
+    const reviewIdCheckPromise = new Promise((resolve,reject) => {
+        if (req.params.review_id.match(/\b\d+\b/g) === null) {
+            reject({status:400,msg:"review ID is not in correct format"})
+        } else {
+            resolve(req.params.review_id)
+        }
+    })
+    return reviewIdCheckPromise.then((review_id) => {
+        return selectReviewsById(review_id)
+    })
```

Comment:
I like the code, but it's not necessary, PSQL will return an error for you if the data type is incorrect. See what happens if you try to get to reviews/banana without the above code. At the stage of the model it should return an error. You should get an error code passed through to your error handling middleware and by checking the code you can fire off a response.